tests/integration/targets/kea_command/tasks/main.yml

```diff
@@ -0,0 +1,138 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+---
+####################################################################
+# WARNING: These are designed specifically for Ansible tests       #
+# and should not be used as examples of how to write Ansible roles #
+####################################################################
+
+# Copyright © Thorsten Glaser <tglaser@b1-systems.de>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+- name: Ensure this is run on the right distro and version the integration test was written for
+  ansible.builtin.assert:
+    that:
+      - ansible_distribution_major_version | int == 13
+      - ansible_distribution | lower == 'debian'
+    fail_msg: "This test was written for a Debian 13 (trixie) container due to uses of its default configuration"
+    quiet: true
```

Comment:
That looks odd to me: it will fail the test unless it is a Debian trixie? Shouldn't it bail out instead?, as in `ansible.builtin.meta: end_play` ?

Reply:
It may or may not fail the test, but the `lineinfile` calls will most likely not find the proper match, and the default socket path is also what is used in Debian (and, to add insult to injury, the socket path’s directory part is not configurable in the config, it depends on how the OS starts the dæmon as it’s in the runtime/pidfile/… directory).

The entire testcase is written to be run on trixie because then I have a known-good state of the KEA services. The module itself will also work elsewhere, but it’s been hard enough to get the server doing what I want for this test.